app/Services/ReportbackService.php

```diff
@@ -67,4 +67,16 @@ public function getReportback($campaignId, $campaignRunId, $userId, $type)
 
         return $reportback ? $reportback : null;
     }
+
+    /*
+     * Handles all business logic around update a reportbackitem(s).
+     *
+     * @param array $data
+     *
+     * @return
+     */
+    public function updateReportbackItems($data)
+    {
+        $this->reportbackRepository->updateReportbackItems($data);
```

Comment:
This would then receive the return value (which you will set from my comment above) and set that to a variable that you pass to the controller like

```
$item = $this->reportbackRepository->updateReportbackItems($data);
return $item
```